quickstarts/github-repo/config.yml

```diff
@@ -0,0 +1,29 @@
+id: 8ea680b5-ced4-4c86-8903-84f7b718fc9f
```

Comment:
Since this is a new quickstart, we need to let the system generate a new id after it is merged into the `main` branch.
```suggestion
```

Reply:
Thanks, I've removed this line.